qtc_packaging/debian_fremantle/control

```diff
@@ -1,4 +1,4 @@
-Source: xbmcremote
```

Comment:
strange. for some reason, github only shows me the removals, not the additions... Sorry for the noise. All fine now. I'll merge this.